live-examples/css-examples/grid/grid-auto-flow.html

```diff
@@ -0,0 +1,43 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="grid-auto-flow">
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">grid-auto-flow: row;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">grid-auto-flow: column;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">grid-auto-flow: sparse;</code></pre>
```

Comment:
I can't find `sparse` as a value in the MDN docs page or the spec, and it doesn't seem to be supported in Firefox or Chrome.

Reply:
you are indeed correct, sparse is the default packing mode, it doesn't actually have a keyword. Some sort of brain autocomplete issue :)

Reply:
and good call on making the changed element a different colour, that really helps.